backend/api/v1/endpoints/chat.py

```
@@ -165,21 +165,40 @@ def get_conversations(db: Session = Depends(get_db), current_user=Depends(get_cu
-    base_dir = os.path.abspath(base)
-    # Use pathlib for safe path joining and resolving
-    try:
-        # Create a safe path by joining and resolving
-        safe_path = os.path.normpath(os.path.join(base_dir, filename))
-        # Verify the path is within the base directory
-        return safe_path if safe_path.startswith(base_dir) else None
-    except (ValueError, TypeError):
-        return None
+# Create a secure file manager for chat uploads
+class SecureFileManager:
+    def __init__(self, directory):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)